ddtrace/propagation/http.py

```diff
@@ -1090,9 +1090,9 @@ def parent_call():
                 headers[_HTTP_BAGGAGE_PREFIX + key] = span_context._baggage[key]
 
         if config._llmobs_enabled:
-            from ddtrace.llmobs._utils import _inject_llmobs_parent_id
+            from ddtrace.llmobs import LLMObs
 
-            _inject_llmobs_parent_id(span_context)
+            LLMObs._inject_llmobs_context(headers)
```

Comment:
Instead of injecting into the headers, let's add it into the context and let the injection mechanisms propagate that into the headers.